src/api/README.md

```diff
@@ -2199,14 +2199,14 @@ Result:
 API sets: `READ`
 
 ```
-URI: /api/v2/notes/notes
-Method: POST
+URI: /api/v2/notes
+Method: GET
 ```
 
 Example:
 
 ```sh
-curl -X POST -H 'Content-Type: application/json' http://127.0.0.1:6420/api/v2/notes/notes
+curl -X GET -H 'Content-Type: application/json' http://127.0.0.1:6420/api/v2/notes
```

Comment:
`-X GET -H 'Content-Type: application/json'` is only needed for the `POST` method